qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -217,30 +217,121 @@ def convert_acquire(self, shift, instruction):
-        """Return converted `AcquireInstruction`, with options to override the qubits,
-        memory_slot, and register_slot fields. This is useful for grouping
-        AcquisitionInstructions which are operated on a single AcquireChannel and
-        a single MemorySlot.
+    def convert_bundled_acquires(
+            self,
+            shift,
+            instructions_,
+    ):
+        """Bundle a list of acquires instructions at the same time into a single
+        Qobj acquire instruction.
 
         Args:
             shift (int): Offset time.
-            instruction (AcquireInstruction): acquire instruction.
-            qubits (list(int)): A list of qubit indices to acquire.
-            memory_slot (list(int)): A list of memory slot indices to store results.
-            register_slot (list(int)): A list of register slot addresses to store results.
+            instructions_ (List[Acquire]): List of acquire instructions to bundle.
         Returns:
             dict: Dictionary of required parameters.
+        Raises:
+            PulseError: If ``instructions`` is empty.
         """
-        res = self.convert_acquire(shift, instruction)
-        if qubits:
-            res.qubits = qubits
-        if memory_slot:
-            res.memory_slot = memory_slot
-        if register_slot:
-            res.register_slot = register_slot
-        return res
+        meas_level = self._run_config.get('meas_level', 2)
+
+        t0 = None
+        duration = None
+        memory_slots = []
+        register_slots = []
+        qubits = []
+        discriminators = []
+        kernels = []
+
+        if not instructions_:
+            raise PulseError('"instructions" may not be empty.')
+
+        for instruction in instructions_:
+            qubits.append(instruction.channel.index)
+
+            instr_t0 = shift + instruction.start_time
+            if t0 is None:
+                t0 = instr_t0
+            elif t0 != instr_t0:
+                raise PulseError(
+                    'Bundled acquire instructions may not have different starting times'
+                )
+
+            instr_duration = instruction.duration
+            if duration is None:
+                duration = instr_duration
+            elif duration != instr_duration:
+                raise PulseError(
+                    'Bundled acquire instructions may not have different starting times'
+                )
+
+            if instruction.mem_slot:
+                memory_slots.append(instruction.mem_slot.index)
+
+            if meas_level == MeasLevel.CLASSIFIED:
+                # setup discriminators
+                if instruction.discriminator:
+                    discriminators.append(
+                        QobjMeasurementOption(
+                            name=instruction.discriminator.name,
+                            params=instruction.discriminator.params,
+                        ),
+                    )
```

Comment:
I rarely see this style in qiskit, try to be consistent with the style that already exists even if it's not your favorite

Reply:
Its [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation). I would argue how I have it is significantly more clear and most importantly consistently indented than:
```python
discriminators.append(QobjMeasurementOption(name=instruction.discriminator.name,
                                                                                  params=instruction.discriminator.params ))
```

Reply:
They're both correct according to PEP 8, so the correct thing to do is be consistent. If you want to use this instead, make a separate PR and fix it everywhere.

Reply:
I also don't understand why you insist on using every possible style that won't simply cause errors. In one PR you use trailing paren, paren on new line unindented one level, and paren on new line in line with the first char on the previous line